code/modules/food&drinks/food/snacks_meat.dm

```diff
@@ -67,6 +67,13 @@
 	trash = /obj/item/trash/plate
 	list_reagents = list("nutriment" = 1, "vitamin" = 4)
 
+/obj/item/weapon/reagent_containers/food/snacks/bearsteak
+	name = "Filet migrawr"
+	desc = "Because eating bear wasn't manly enough."
+	icon_state = "bearsteak"
+	trash = /obj/item/trash/plate
+	list_reagents = list("nutriment" = 2, "vitamin" = 5, "manlydwarf" = 5)
```

Comment:
manlydorf instead of manlydwarf. Also, it's not entirely necessary as the snack inherits the reagents that were necessary to construct the recipe.